report_qweb_decimal_place/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,32 @@
+This module intends to provide the base function for currencies to adjust the number of decimal places
+for the unit price in QWeb reports.
+Installing this module alone does not affect the presentation of existing QWeb reports.
+Individual adjustments need to be done in separate modules in a manner similar to the following:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <odoo>
+        <template
+            id="report_saleorder_document_inherit"
+            inherit_id="sale.report_saleorder_document"
+        >
+            <xpath expr="//tbody[@class='sale_tbody']//tr//td[3]" position="replace">
+                <td name="td_priceunit" class="text-right">
+                    <t t-set="currency" t-value="doc.currency_id" />
+                    <t t-set="price_unit" t-value="line.price_unit" />
+                    <t t-call="report_qweb_decimal_place.price_unit_value_format" />
+                </td>
+            </xpath>
+        </template>
+    </odoo>
```

Comment:
```suggestion
    <template
        id="report_saleorder_document_inherit"
        inherit_id="sale.report_saleorder_document"
    >
        <xpath expr="//td[@name='td_priceunit']/span" position="replace">
                <t t-set="currency" t-value="doc.currency_id" />
                <t t-set="price_unit" t-value="line.price_unit" />
                <t t-call="report_qweb_decimal_place.price_unit_value_format" />
        </xpath>
    </template>
```
Please double-check if this suggestion is valid.

Reply:
Yes. This suggestion is valid.